lib/cube_browser/__init__.py

```diff
@@ -431,7 +431,7 @@ class Contourf(Plot2D):
     """
     def draw(self, cube):
         self.element = iplt.contourf(cube, axes=self.axes, coords=self.coords,
-                                     **self.kwargs)
+                                     extend='both', **self.kwargs)
```

Comment:
@heatherguy Thanks for the PR!

Could you elaborate on the artifacts that are being removed by adding `extend='both'`?

My worry is that this is papering over other incorrect behaviour, which should be investigated and corrected.

Also, using `extend='both'` here is forcing a default that all users may not want or desire, so we need to be careful.

Note that, `extend='both'` can also be passed through from top API level e.g. `Contourf(..., extend='both')`